pyopenms_viz/_config.py

```diff
@@ -362,6 +362,15 @@ class SpectrumConfig(VLineConfig):
     mirror_spectrum: bool = False
     peak_color: str | None = None
 
+    # New fields for peptide sequence plotting
+    display_peptide_sequence: bool = False
+    peptide_sequence: str = ""
+    matched_fragments: list[tuple] = field(default_factory=list)
+    peptide_sequence_fontsize: int = 12
+    peptide_sequence_color: str = "red"
+    highlight_color: str = "blue"
+    highlight_alpha: float = 0.8
```

Comment:
Please see comments below, some of these arguments might not be needed because they are expected to be specified in the dataframe. (lines 365- 372)